patches/chromium/preconnect_manager.patch

```diff
@@ -10,13 +10,13 @@ in favor of defining PreconnectRequest in this file since we don't build
 the header.
 
 diff --git a/chrome/browser/predictors/preconnect_manager.cc b/chrome/browser/predictors/preconnect_manager.cc
-index 7eb87e7e7128da73ff78323c50d4efa488de781c..54cd08ea761076034fda450abe07e40d3235d9b0 100644
+index 02bdfd1f176d4ce44984387226b6fc2e6551f15e..073c505c31d5daa08d3582a4739ef8703552fd6e 100644
 --- a/chrome/browser/predictors/preconnect_manager.cc
 +++ b/chrome/browser/predictors/preconnect_manager.cc
-@@ -11,9 +11,11 @@
- #include "base/metrics/histogram_macros.h"
+@@ -13,9 +13,11 @@
  #include "base/not_fatal_until.h"
  #include "base/trace_event/trace_event.h"
+ #include "chrome/browser/predictors/predictors_features.h"
```

Comment:
I wasn't 100% sure whether to include the new changes that added predictors_features to this file. Here is the CL: https://chromium-review.googlesource.com/c/chromium/src/+/5725672

Our patch ignores the other predictor related code but I figured extra code to limit the number of preconnect sockets to one can still be useful even if Electron doesn't use resource_prefetch_predictor, so I left it in. Let me know otherwise!

Reply:
@alicelovescake if it isn't causing problems, it is fine to leave it as is in the patch.